pkg/adapter/anchore/credential/defaultloader.go

```diff
@@ -0,0 +1,7 @@
+package credential
+
+type DefaultCredenitalLoader struct{}
+
+func (c *DefaultCredenitalLoader) LoadFromCredentialStore(passwordConfig string) string {
```

Comment:
I'd like to make the secret/cred configuration more explicit rather than an encoded value in the password, but I think we can start with this as a draft and follow-up with another commit to change that behavior before a release is pushed

Reply:
Sure, we can talk after Feb 3 as I will be in Chinese new year holiday from next week. I'd like to hear more ideas from you ;-)

Reply:
Have a great new year holiday! We'll sync up when you return.